azure-devops/create-1es-hosted-pool.ps1

```diff
@@ -140,66 +140,21 @@ New-AzResourceGroup `
   -Location $Location `
   -Tag $SimplySecureV2OptInTag | Out-Null
 
+####################################################################################################
+Display-ProgressBar -Status 'Creating credentials'
+
+$AdminPW = New-Password
 $AdminPWSecure = ConvertTo-SecureString $AdminPW -AsPlainText -Force
 $Credential = New-Object System.Management.Automation.PSCredential ('AdminUser', $AdminPWSecure)
 
 ####################################################################################################
 Display-ProgressBar -Status 'Creating virtual network'
 
-$allowHttp = New-AzNetworkSecurityRuleConfig `
-  -Name AllowHTTP `
-  -Description 'Allow HTTP(S)' `
-  -Access Allow `
-  -Protocol Tcp `
-  -Direction Outbound `
-  -Priority 1000 `
-  -SourceAddressPrefix * `
-  -SourcePortRange * `
-  -DestinationAddressPrefix * `
-  -DestinationPortRange @(80, 443)
-
-$allowQuic = New-AzNetworkSecurityRuleConfig `
-  -Name AllowQUIC `
-  -Description 'Allow QUIC' `
-  -Access Allow `
-  -Protocol Udp `
-  -Direction Outbound `
-  -Priority 1010 `
-  -SourceAddressPrefix * `
-  -SourcePortRange * `
-  -DestinationAddressPrefix * `
-  -DestinationPortRange 443
-
-$allowDns = New-AzNetworkSecurityRuleConfig `
-  -Name AllowDNS `
-  -Description 'Allow DNS' `
-  -Access Allow `
-  -Protocol * `
-  -Direction Outbound `
-  -Priority 1020 `
-  -SourceAddressPrefix * `
-  -SourcePortRange * `
-  -DestinationAddressPrefix * `
-  -DestinationPortRange 53
-
-$denyEverythingElse = New-AzNetworkSecurityRuleConfig `

```

Comment:
It's unfortunate that this rule which was intended to prevent turning the CI into a botnet has no effect on the VM image. Should we contact 1ES folks and ask if they have any recommendations for things we can/should do to secure the VMs given that we're literally compiling and running code submitted by anyone with a Github account? (No change requested.)

Reply:
I'll follow up offline.